### PR TITLE
Add MySQL support to certdb

### PR DESCRIPTION
**Pull request by drcapulet**

### CODE REVIEW

**Comment by drcapulet** [comment]
Ping @kisom / @lziest

File: vendor/github.com/go-sql-driver/mysql/LICENSE [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2014 CloudFlare Inc.
```
Comment (kisom) [inline vendor/github.com/go-sql-driver/mysql/LICENSE]: Is this the right LICENSE file?

**Comment by kisom** [comment]
@drcapulet just an FYI, currently reviewing this. It's taking a while as there's a lot of vendored code to read through.

File: vendor/github.com/go-sql-driver/mysql/LICENSE (hunk 1 above)
Comment (drcapulet) [inline vendor/github.com/go-sql-driver/mysql/LICENSE]: Nice catch, I updated `gvt` and it solved the problem.

**Comment by drcapulet** [comment]
@kisom Awesome, thanks!

**Comment by kisom** [comment]
LGTM,.

**Comment by deanefrati** [comment]
any idea when this will be merged? we would really like to see cfssl support mysql since it will be easier to stand up a clustered implementation this way using mariadb.

**Comment by lziest** [comment]
LGTM, so the SQL statements we put down works with mysql as well? I am happy and feeling unreal.

**Comment by lziest** [comment]
still have conflicts though

**Comment by kisom** [comment]
@deanefrati @drcapulet Once this is rebased, it can be merged.

**Comment by drcapulet** [comment]
@kisom / @lziest rebased, should be good now! The SQL statements work due to the changes a little while back to use sqlx since it knows how to handle different drivers - in theory you could also add Oracle support fairly quickly as well.

**Comment by kisom** [comment]
Thanks!

**Pull request merged**